### ISSUE TITLE
Enhancement : Regarding mp4parser version

### ISSUE THREAD

**Issue by AmanSharma1988** [Tourenathan-G5organisation/SiliCompressor#23]
Right now we are using version : 1.0.6 and we use below for movie
**Movie movie = MovieCreator.build((DataSource) randomAccessFile.getChannel());**

I checked below gradle link has higher version.
compile 'com.googlecode.mp4parser:isoparser:1.1.20'
And big difference is this new version. In this we have below:
**Movie movie = MovieCreator.build(new FileDataSourceViaHeapImp(src.getAbsolutePath()));**
**// NOTE: Switched to using FileDataSourceViaHeapImpl since it does not use memory mapping (VM).
// Otherwise we get OOM with large movie files.**

**Comment by gzgang** [comment]
Hi, when I use version 1.1.22; after compress a video, the output video file can't play.
did you have this problem ?
I use VLC player played the output video, it's tip me  can't find decoder for audio/mp4a-latm

**Comment by developerGM** [comment]
Yes, for now, SiliCompressor can works only with mp4parser 1.0.6

**Comment by AmanCyber** [comment]
Did anyone got OOM with large file?

**Comment by tiagoalexbastos** [comment]
any updates on this ?
